exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/lateraljoin/TestE2EUnnestAndLateral.java

```diff
@@ -656,4 +656,29 @@ public void testUnnestNestedStarSubquery() throws Exception {
         .go();
   }
 
+  @Test
```

Comment:
Can you add a test case with multi lateral.

Reply:
Added `testMultiLateralWithComplexProject`.